lambda/fetchDeviceShadow.ts

```diff
@@ -56,7 +56,7 @@ const connectionsRepo = connectionsRepository(
 	db,
 	websocketDeviceConnectionsTableName,
 )
-const deviceShadowPromise = (async () => {
+const deviceShadowPromise = once(async () => {
```

Comment:
Unrelated change.

Reply:
This effectively does the same thing as before. It ensures that deviceShadowPromise only exists once. It will stay in a rejected stated if parameters cannot be fetched.

Reply:
Ah, right. Now I see. It does fix the problem, because the lambda is invoked only in the handler.